src/leveldbwrapper.h

```diff
@@ -168,6 +199,44 @@ class CLevelDBWrapper
     {
         return pdb->NewIterator(iteroptions);
     }
+
+    /**
+     * Return true if the database managed by this class contains no entries.
+     */
+    bool IsEmpty()
+    {
+        leveldb::Iterator* it = NewIterator();
+        it->SeekToFirst();
+        bool empty = !(it->Valid());
+        delete it;
+        return empty;
+    }
+
+    /**
+     * Accessor for obfuscate_key.
+     */
+    std::string ObfuscateKey() const { return obfuscate_key; }
```

Comment:
I'd make this signature `std::string &GetObfuscateKey()`

- Otherwise it looks like an action: Obfuscate the key!
- As this returns a field of the object, we can avoid a copy by returning a const reference